### PR TITLE
fix UI and add App icon

### PR DESCRIPTION
**Pull request by Tahia50**

### CODE REVIEW

**Comment by dina28** [comment]
ok 👍

File: FileReaderApp/Base.lproj/Main.storyboard [hunk 1]
```diff
@@ -18,37 +18,65 @@
...
                                     <action selector="showPPT:" destination="BYZ-38-t0r" eventType="touchUpInside" id="YnH-I9-Vh3"/>
                                 </connections>
                             </button>
+                            <button opaque="NO" contentMode="scaleToFill" contentHorizontalAlignment="center" contentVerticalAlignment="center" lineBreakMode="middleTruncation" translatesAutoresizingMaskIntoConstraints="NO" id="pql-T7-5ZF">
+                                <rect key="frame" x="125" y="132" width="350" height="27"/>
+                                <constraints>
+                                    <constraint firstAttribute="height" relation="greaterThanOrEqual" constant="27" id="XdB-ja-bRB"/>
+                                    <constraint firstAttribute="height" constant="27" id="dVB-Pg-QvB"/>
+                                    <constraint firstAttribute="width" constant="350" id="dYg-wm-2A2"/>
+                                    <constraint firstAttribute="width" relation="greaterThanOrEqual" constant="350" id="xsW-zM-HIp"/>
```
Comment (hasanmainul) [inline FileReaderApp/Base.lproj/Main.storyboard]: redundant constraint

File: FileReaderApp/Base.lproj/Main.storyboard [hunk 2]
```diff
@@ -18,37 +18,65 @@
...
+                                    <color key="titleColor" red="1" green="1" blue="1" alpha="1" colorSpace="calibratedRGB"/>
+                                </state>
                                 <connections>
                                     <action selector="showPPT:" destination="BYZ-38-t0r" eventType="touchUpInside" id="YnH-I9-Vh3"/>
                                 </connections>
                             </button>
+                            <button opaque="NO" contentMode="scaleToFill" contentHorizontalAlignment="center" contentVerticalAlignment="center" lineBreakMode="middleTruncation" translatesAutoresizingMaskIntoConstraints="NO" id="pql-T7-5ZF">
+                                <rect key="frame" x="125" y="132" width="350" height="27"/>
+                                <constraints>
+                                    <constraint firstAttribute="height" relation="greaterThanOrEqual" constant="27" id="XdB-ja-bRB"/>
```
Comment (hasanmainul) [inline FileReaderApp/Base.lproj/Main.storyboard]: redundant constraint

**Comment by hasanmainul** [comment]
LGTM 👍

**Pull request merged**